### PR TITLE
Fix DefaultActiveSpeakerPolicy init on Xcode 12

### PR DESCRIPTION
**Pull request by zhinang-amazon**
### Issue #, if available:
When running Swift Demo with SDK build as binary in Xcode 12, it crashes accessing `DefaultActiveSpeakerPolicy` init with no param, and this is also not accessible in ObjC

Added init with no param explicitly, and use in obj c demo app
### Testing done:
Swift demo does not crash anymore on Xcode 12
Obj C demo works as expected and gets the ActiveSpeakerObserver callbacks

#### Manual test cases (add more as needed):

- [X] Join meeting without CallKit
- [ ] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [X] Leave meeting
- [X] Rejoin meeting
- [ ] See metrics
- [ ] See attendee presence
- [X] Send audio
- [X] Receive audio
- [] Mute self
- [ ] Unmute self
- [ ] See local mute indicator when muted
- [ ] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [ ] Enable and disable remote video from remote side
- [X] Send local video
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share

#### Integration validation (required before release)

- [X] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [X] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [X] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [X] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [X] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [X] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerpolicy/DefaultActiveSpeakerPolicy.swift [hunk 1]
```diff
@@ -9,19 +9,28 @@
 import Foundation
 
 @objcMembers public class DefaultActiveSpeakerPolicy: NSObject, ActiveSpeakerPolicy {
+    public static let defaultSpeakerWeight = 0.9
```
Comment (hokyungh) [inline AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerpolicy/DefaultActiveSpeakerPolicy.swift:12]: Few questions, in swift, can't they specify which to input? For instance, they can set weight but sets others as default right? Wouldn't this break that case?

Comment (zhinang-amazon) [reply AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerpolicy/DefaultActiveSpeakerPolicy.swift:12]: They are still use the other init with all three params

**Review by hokyungh — DISMISSED** [review]

File: AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/ViewControllerObjC.m [hunk 2]
```diff
@@ -360,4 +369,16 @@ - (void)videoTileSizeDidChangeWithTileState:(VideoTileState *)tileState {
     [self.logger infoWithMsg:[NSString stringWithFormat:@"Video Tile size changed: tileId: %ld, attendeeId: %@", (long)tileState.tileId, tileState.attendeeId]];
 }
 
+# pragma mark - ActiveSpeakerObserver
+
+- (void)activeSpeakerScoreDidChangeWithScores:(NSDictionary<AttendeeInfo *, NSNumber *> * _Nonnull)scores {
+    [self.logger infoWithMsg:@"activeSpeakerScoreDidChangeWithScores callback invoked"];
```
Comment (tianyux-amz) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/ViewControllerObjC.m:375]: Have you checked if this callback was invoked? I think it won't be invoked unless you specified `scoresCallbackIntervalMs`. Ref: https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerdetector/ActiveSpeakerObserver.swift#L20

**Review by tianyux-amz — COMMENTED** [review]
Could you provide some background on why it stops working on Xcode 12?

**Comment by zhinang-amazon** [comment]
> Could you provide some background on why it stops working on Xcode 12?

Without explicitly adding init(), swift demo crashes when initializing `DefaultActiveSpeakerPolicy`
`Fatal error: Use of unimplemented initializer 'init()' for class 'AmazonChimeSDK.DefaultActiveSpeakerPolicy'`

I have not found any documentation or bug report on why this only happens on Xcode 12, when it's from a framework as binary. I'm guessing Xcode 12 is still fairly new.

File: AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/ViewControllerObjC.m (hunk 2 above)
Comment (zhinang-amazon) [reply AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/ViewControllerObjC.m:375]: You are right. Implemented scoresCallbackIntervalMs and verified it's getting called

**Review by tianyux-amz — APPROVED** [review]

**Review by hokyungh — APPROVED** [review]

**Pull request merged**